### PR TITLE
Clean db namespace and remove its reference usage - Closes #2632

### LINKED ISSUE #2632: Clean db namespace and remove its reference usage
Scan all modules to find interfaces which can be replaced by `storage.entities`

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?
We had removed the usage of `db` namespace from the application, and there are some left over suspended in the code. 

### How did I fix it?
Removed all the references of `db` in the application and fixed tests. 

### How to test it?
Run all tests. 

### Review checklist

* The PR resolves #2632
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/common/modules_loader.js [hunk 1]
```diff
@@ -116,7 +110,7 @@ const modulesLoader = new function() {
 				async.waterfall(
 					[
 						function(waterCb) {
-							return new Account(scope.db, scope.schema, scope.logger, waterCb);
+							return new Account(scope.schema, scope.logger, waterCb);
```
[the remark this answers is not in the document]
Comment (pablitovicente) [reply test/common/modules_loader.js]: Oh this was old code we just tried to limit the scope of the changes. The refactor was only for removing the DB and replace with storage.

Other refactors I think should be part of a different issue.

File: test/common/application.js [hunk 2]
```diff
@@ -59,507 +56,479 @@ function __init(initScope, done) {
...
+							return dns.lookup(peer.ip, { family: 4 }, (err, address) => {
+								if (err) {
+									console.error(
+										`Failed to resolve peer domain name ${
+											peer.ip
+										} to an IP address`
+									);
+									return callback(err, peer);
+								}
+								return callback(null, Object.assign({}, peer, { ip: address }));
```
[the remark this answers is not in the document]
Comment (pablitovicente) [reply test/common/application.js]: Oh this was old code we just tried to limit the scope of the changes. The refactor was only for removing the DB and replace with storage.

Other refactors I think should be part of a different issue.

File: test/common/modules_loader.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (pablitovicente) [reply test/common/modules_loader.js]: Already fixed. Resolving.

File: test/common/application.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (limiaspasdaniel) [reply test/common/application.js]: Oh ok! Didn't know :) I agree.

**Pull request merged**